README.md

```diff
@@ -19,6 +47,54 @@ For formatting code automatically please use `poetry run autopep8 --in-place --r
+  - ... [vendors](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/mach_vendor.py#L42-L43) the latest version from upstream by calling `./mach vendor manifest path/to/moz.yaml` ...
+  - ... [checks the result](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L159-L171) ...
+  - ... [commits](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/hg.py#L14-L19) the change locally...
+  - ... [submits it to try](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/apis/taskcluster.py#L51) ...
+  - ... [comments on the bug](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L179) ...
+  - ... and (finally) [adds it to the Updatebot database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L181).
+- That database lives in Google CloudSQL.  There is a dev and prod database, as well as [dev and prod credentials](https://searchfox.org/mozilla-central/rev/83e67336083df9f9a3d1e0c33f2ba19703d57161/taskcluster/docker/updatebot/run.py#79-84) for those databases, bugzilla, try server, phabricator, sentry, and sql-proxy (which is used to connect to the database).  You can find them in [grants.yml](https://hg.mozilla.org/ci/ci-configuration/file/tip/grants.yml#l644) searching for 'updatebot'.  The dev creds are granted to holly, which is our reserved development instance because we can't test it o try.  The prod creds are only available on mozilla-central.
+- The next time the updatebot job runs it will get to step (6) and see that it has seen the (new) revision before.  It will [process the job from there](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L238).
+  1. Will check if all the jobs in the try run are done. If they are not, it will do nothing and check again on the next updatebot run.
+  2. If they are done, it will look to see if there any test failures. If so it will [retrigger them](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L384) and wait until the next job. For the sake of explanation we [will continue as if we've gotten the trigger results](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L404) 
+  3. We [look at the failures](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L411-L431), summarize them, [add a comment to the bug and update the database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L420-L431).
```

Comment:
`For the sake of explanation we [will continue as if we've gotten the trigger results](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L404) 3. We [look at the failures](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L411-L431), summarize them, [add a comment to the bug and update the database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L420-L431).`
->
`3. Once we've [received the retrigger results](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L404) then we [look at failures)(https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L411-L431), summarize them, and [add a comment to the bug and update the database](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/tasktypes/vendoring.py#L420-L431).`